File: abc/abc222/b.java
// https://atcoder.jp/contests/abc222/submissions/43222375

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int n = stdIn.nextInt();
    int p = stdIn.nextInt();

    int[] a = new int[n];
    for(int i = 0; i < n; i++) a[i] = stdIn.nextInt();

    int ans = 0;
    for(int x : a) {
      if(x < p) {
        ans++;
      }
    }

    System.out.println(ans);
  }
}

File: abc/abc222/b.cs
// https://atcoder.jp/contests/abc222/submissions/46506127

using System;

public class A
{
  static void Main()
  {
    string[] t = Console.ReadLine().Split(" ");
    int n = int.Parse(t[0]);
    int p = int.Parse(t[1]);
    string[] a = Console.ReadLine().Split(" ");

    int ans = 0;
    for(int i = 0; i < n; i++)
    {
      if(int.Parse(a[i]) < p)
      {
        ans++;
      }
    }

    Console.Write(ans);
  }
}

